modules/java_api/tests/InferRequestTests.java

```diff
@@ -25,34 +24,34 @@ public void setUp() {
         completionCallback = false;
     }
 
-    @Ignore
     @Test
     public void testGetPerformanceCounts() {
         inferRequest.Infer();
 
         Vector<String> layer_name = new Vector<>();
-        layer_name.add("19/Fused_Add_");
+        layer_name.add("19/WithoutBiases");
+        layer_name.add("20/mean/Fused_Mul_614616_const");
+        layer_name.add("20/mean/Fused_Mul_614616_const_abcd_Acdb16a_19/WithoutBiases");
```

Comment:
This is unstable check. The values are generated differently between runs and machines.